src/util/hidepassword.ts

```diff
@@ -0,0 +1,14 @@
+export class HidePassword {
```

Comment:
HidePassword.hideTokenpassword is not a good name.  
Why not just export two functions instead without class at all:
* filterPassword
* filterToken

name this file as `filters.ts`.

Then in code it would look like:

``` JavaScript
import * as Filters from './utils/filters';

Filters.filterPassword(string);
```